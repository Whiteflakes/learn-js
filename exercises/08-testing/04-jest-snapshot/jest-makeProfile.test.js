const { makeProfile } = require('./index');

test('makeProfile snapshot', () => {
  const p = makeProfile('Alice', 30);
  // Normalize the id so snapshot is stable
  p.id = 'alice-TS';
  expect(p).toMatchSnapshot();
});
