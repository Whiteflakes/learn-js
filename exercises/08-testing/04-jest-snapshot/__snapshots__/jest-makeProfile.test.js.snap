// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`makeProfile snapshot 1`] = `
{
  "age": 30,
  "id": "alice-TS",
  "name": "Alice",
  "settings": {
    "notifications": {
      "email": true,
      "sms": false,
    },
    "theme": "dark",
  },
  "tags": [
    "student",
    "learner",
  ],
}
`;
